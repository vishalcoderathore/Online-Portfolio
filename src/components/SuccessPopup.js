import React from 'react';

class SuccessPopup extends React.Component {
    render() {
        return (
            <div className="form__success">
                <h5>
                    Success
                </h5>
                <p>Your message has been delivered!</p>
            </div>
        )
    }
}

export default SuccessPopup;