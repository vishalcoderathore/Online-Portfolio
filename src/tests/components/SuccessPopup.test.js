import React from 'react';
import { shallow } from 'enzyme';
import SuccessPopup from '../../components/SuccessPopup';

test('should render Loading Page correctly', () => {
    const wrapper = shallow(<SuccessPopup />);
    expect(wrapper).toMatchSnapshot();
});