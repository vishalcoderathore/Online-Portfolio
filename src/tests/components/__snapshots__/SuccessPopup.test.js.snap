// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading Page correctly 1`] = `
<div
  className="form__success"
>
  <h5>
    Success
  </h5>
  <p>
    Your message has been delivered!
  </p>
</div>
`;
